docs/source/get-involved.rst

```diff
@@ -0,0 +1,46 @@
+Get involved
+============
```

Comment:
Great section to have.  If possible I'd like for things to be in the following order:

1.  Stackoverflow
2.  Github issues
3.  Email list
4.  Gitter chat

This is the order in which we want people to use these resources.  The earlier ones have much more reuse value than the latter, in which we end up answering very similar questions repeatedly.